modules/friendlyshared.js

```diff
@@ -16,15 +16,18 @@
 Twinkle.shared = function friendlyshared() {
 	if( mw.config.get('wgNamespaceNumber') === 3 && Morebits.isIPAddress(mw.config.get('wgTitle')) ) {
 		var username = mw.config.get('wgTitle').split( '/' )[0].replace( /\"/, "\\\""); // only first part before any slashes
-		Twinkle.addPortletLink( function(){ Twinkle.shared.callback(username); }, "Shared IP", "friendly-shared", "Shared IP tagging" );
+		Twinkle.addPortletLink( function(){ Twinkle.shared.callback(username); }, $.i18n._('tw-sharedip-label'), "friendly-shared", 'tw-sharedip-label-hover' );
 	}
 };
 
 Twinkle.shared.callback = function friendlysharedCallback( uid ) {
+	console.log(Twinkle.shared.standardList);
 	var Window = new Morebits.simpleWindow( 600, 420 );
-	Window.setTitle( "Shared IP address tagging" );
-	Window.setScriptName( "Twinkle" );
-	Window.addFooterLink( "Twinkle help", "WP:TW/DOC#shared" );
+	Window.setTitle( $.i18n._('tw-sharedip-window-title') );
+	Window.setScriptName( $.i18n._('tw-core-script-name') );
+	Window.addFooterLink( $.i18n._('tw-core-help'), $.i18n._('tw-core-help-page') );
```

Comment:
Unsure what you mean. If the purpose is to let them replace the footer link with something else, it is already possible (they need to override the values for `tw-core-help` and `tw-core-help-page`). If you mean they should be able to have more than one footer link if they deem necessary, then that is an issue beyond the scope of this change. I am trying not to add or remove any functionality, but simply make them i18nized and l10inzed exactly as they are.